zrp/prepare/prepare.py

```diff
@@ -57,7 +57,6 @@ def transform(self, input_data):
             Dataframe to be transformed
```

Comment:
Incorrect parameter provided in the fit

Reply:
Parameter data is used in lines 38 and 46. Changed the parameter name to input_data so the name is the same as in transform method.